config/config.go

```diff
@@ -732,8 +732,6 @@ func (cfg *FastSyncConfig) ValidateBasic() error {
 	switch cfg.Version {
 	case "v0":
 		return nil
-	case "v1":
```

Comment:
some people use v1.

Reply:
Are we keeping v1 into 0.34? I thought we decided to remove it in the next major release?

Reply:
even if so, can this be a separate PR?

Reply:
https://github.com/tendermint/tendermint/pull/4656, made it into a separate PR.